content/guides/caching.md

```diff
@@ -4,284 +4,358 @@ sort: 11
 contributors:
   - okonet
   - jouni-kantola
+  - skipjack
 related:
   - title: Predictable Long Term Caching
     url: https://medium.com/webpack/predictable-long-term-caching-with-webpack-d3eee1d3fa31
+  - title: Long Term Caching of Static Assets
+    url: https://medium.com/@okonetchnikov/long-term-caching-of-static-assets-with-webpack-1ecb139adb95#.vtwnssps4
+  - title: Webpack & Caching
+    url: https://gist.github.com/sokra/ff1b0290282bfa2c037bdb6dcca1a7aa
+  - title: Advanced Webpack Presentation
+    url: https://presentations.survivejs.com/advanced-webpack/
+  - title: Issue 1315
+    url: https://github.com/webpack/webpack/issues/1315
+  - title: Issue 652
+    url: https://github.com/webpack/webpack.js.org/issues/652
 ---
 
-To enable long-term caching of static resources produced by webpack:
+T> This examples in this guide stem from [getting started](/guides/getting-started), [output management](/guides/output-management) and [code splitting](/guides/code-splitting).
 
-1. Use `[chunkhash]` to add a content-dependent cache-buster to each file.
-2. Extract the webpack manifest into a separate file.
-3. Ensure that the entry point chunk containing the bootstrapping code doesn’t change hash over time for the same set of dependencies.
+So we're using webpack to bundle our modular application, deploying our `/dist` directory to the server, and clients, typically browsers, are hitting that server to grab the site and its assets. The last step can be time consuming, which is why browsers use a technique called [caching](https://en.wikipedia.org/wiki/Cache_(computing)). This allows sites to load faster with less unnecessary network traffic, however it can also cause headaches when you need new code to be picked up.
```

Comment:
I'm not sure this guide needs this intro. It is good for a blog post but I'd expect this guide to get directly to the point. Probably replace it with a link to [my post](https://medium.com/@okonetchnikov/long-term-caching-of-static-assets-with-webpack-1ecb139adb95) and I could cross-link to this guide in the beginning to guide the traffic here?